server/test/check.go

```diff
@@ -1205,6 +1205,34 @@ func TestCheckQuery(t *testing.T, datastore storage.OpenFGADatastore) {
 				Allowed: true,
 			},
 		},
+		{
+			name:             "EdgeCase1",
+			resolveNodeLimit: defaultResolveNodeLimit,
+			request: &openfgapb.CheckRequest{
+				TupleKey: tuple.NewTupleKey("document:1", "viewer", "abigail"),
+			},
+			typeDefinitions: []*openfgapb.TypeDefinition{
+				{
```

Comment:
Add the `user` type definition so it's not ambiguous.

Reply:
Done! Thanks.